pkg/apis/openebs.io/v1alpha1/cas_keys.go

```diff
@@ -76,6 +63,36 @@ const (
 	StorageClassHeaderKey CASKey = "storageclass"
 )
 
+// CASSimpleKey represents a openebs key used either in resource annotation 
+// or label
+//
+// NOTE:
+//  SimpleKey (i.e. without 'openebs.io/' ) helps to parse key via 
+// go templating
+type CASSimpleKey string
```

Comment:
I think it can be renamed in a better way. Any strong reasons for using `Simple`? How does `CASPlainKey` sounds?

Reply:
Plain reflects better @princerachit ... Will get it done...thanks